pkg/rest/rest.go

```diff
@@ -0,0 +1,194 @@
+package rest
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/julienschmidt/httprouter"
+	"github.com/vinted/rest-dhcpd/pkg/configdb"
+	"log"
+	"net/http"
+	//"sync"
+)
+
+func index(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	fmt.Fprintf(w, "<html><body>REST-DHCPD<br>https://github.com/vinted/rest-dhcpd</body></html>")
+}
+
+func client_config_show(w http.ResponseWriter, r *http.Request, params httprouter.Params) {
```

Comment:
I think `camelCase` is needed by Go conventions to make linters happy in the future

Reply:
thank you!